chapters/git-cmdline.Rmd

```diff
@@ -13,11 +13,11 @@ Originally developed to support software development,
 over the past fifteen years it has become the cornerstone
 of \gref{reproducible research}{reproducible_research}.\index{reproducible research!version control}
 
-A version control system stores a master copy of your code in a repository,
+The system works by storing a master copy of your code in a repository,
```

Comment:
How about "Version control works" instead of "The system works" (more specific subject)